components/addresses/index.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable react/no-unused-state */
+import React, { Component } from 'react';
+import { View } from 'react-native';
+import SearchBar from '../searchBar';
+import SearchBarDestination from '../searchBarDestination';
+import BackButton from './backButton';
+import CurrentLocation from './currentLocation';
+import Destination from './destination';
+import Car from './car';
+import Bus from './bus';
+import Walking from './walking';
+import Bike from './bike';
+import styles from './styles';
+
+export default class Addresses extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      region: {
+        latitude: 0,
+        longitude: 0,
+        latitudeDelta: 0.05,
+        longitudeDelta: 0.05
+      },
+      hide: true,
+    };
+  }
+
+    updateRegion = (newRegion) => {
+      this.setState({
+        region: {
+          latitude: newRegion.latitude,
+          longitude: newRegion.longitude,
+          latitudeDelta: 0.05,
+          longitudeDelta: 0.05
+        }
+      });
+      this.props.getRegion(newRegion);
+    };
+
+
+    updateCoordinates = (newCoordinates) => {
+      this.setState({
+        coordinates: newCoordinates
+      });
+      this.props.getCoordinates(newCoordinates);
+    };
+
+    getPolylinePoint = (data) => {
+      this.setState({
+        encryptedLine: data
+      });
+    };
+
+    render() {
+      return (
+        <View style={styles.searchContainer}>
+          <SearchBar updateRegion={this.updateRegion} hideMenu={this.state.hide} />
+          <SearchBarDestination
```

Comment:
The placeholder for the translation is not given here
![IMG_0010](https://user-images.githubusercontent.com/30513164/76670345-6859b300-6566-11ea-80cc-4f656783fa28.PNG)

Reply:
@Nichita-Hariton  Thank you for notifying us, we will be doing the required changes

Reply:
This translation has been changed however I see another one missing: It is the one for the default start location which is "My current Location" this is not translated as well. Please Fix

Reply:
The problem has been fixed